### ISSUE TITLE
release 0.9.0 plan

### ISSUE THREAD

**Issue by zhenhuadong** [kubedb/project#281]
I have two question on 0.9.0 release

1. When is the target date of 0.9.0 release? 

2. Could you confirm that the following features (or bug fix) will be in release 0.9.0?
-  support rolling update to new postgres version (minor version update) 
-  Synchronous Replication in PostgreSQL operator (https://github.com/kubedb/project/issues/251)
-  Initialization from Snapshot for local backend fail (https://github.com/kubedb/project/issues/267)
-  Snapshot cannot be removed for `hostPath` backend (https://github.com/kubedb/project/issues/261)

**Comment by hossainemruz** [comment]
@zhenhuadong 
```
When is the target date of 0.9.0 release?
```
We are planning to cut a release early in the next week.

```
support rolling update to new postgres version (minor version update)
```
Next release will not support automatic rolling update to new postgres version. However, we are introducing a new `PostgresVersion` crd that will allow you to use a custom image.
```
Synchronous Replication in PostgreSQL operator
```
Currently, we are working on some other critical issues. Once we are done, we will review your PR. This might not be available in next release. We were having some problems with synchronous replication in the past. We must ensure that everything works well now.
```
Initialization from Snapshot for local backend fail
``` 
We are already working on this. This will be fixed in 0.9.
```
Snapshot cannot be removed for hostPath backend
```
We have already this fixed in a pending PR. This will be available in 0.9. However,  you will able to remove snapshot for local backend but it will not remove respective data from hostPath unlike other backends.

**Comment by tamalsaha** [comment]
0.9.0-rc.0 released. This should be usable now in production systems. We are going to close out remaining issues in 0.9.0 milestone in the coming weeks.
